clowdapp.yaml

```diff
@@ -7,7 +7,7 @@ objects:
 - apiVersion: cloud.redhat.com/v1alpha1
   kind: ClowdJobInvocation
   metadata:
-    name: compliance
+    name: compliance-import-ssg-${RANDOM_SUFFIX}
```

Comment:
This change isn't related to this PR. We're going to be moving this to a CronJob and removing this CJI in RHICOMPL-2145. If you want to add that work to this PR in another commit, I wouldn't mind :) Otherwise, we can merge just the remediations change and create a new PR for RHICOMPL-2145.

Reply:
The CJI is not meant to be used as we're using it now in this template. The way is created, once deployed, you'll never be able to update it ever. A CJI is a one time job run. In order to be run every time, it requires to have a different name, hence the dynamic suffix. However it is true that this changes is maybe outside of this PR, and will be useless when the CJI is deprecated. I'll remove it from the final version.

Reply:
Technically there would be an image change on a deployment which would likely trigger the another one-time job, wouldn't it?

Reply:
nope, since the CJI has the same name, there would not be another deployment as that resource is already there. It must have different names.